modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v3_7_1/test/DDMStructureVersionDDMStructureCTUpgradeProcessTest.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.upgrade.v3_7_1.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.dynamic.data.mapping.io.DDMFormSerializer;
+import com.liferay.dynamic.data.mapping.model.DDMStructure;
+import com.liferay.dynamic.data.mapping.model.DDMStructureVersion;
+import com.liferay.dynamic.data.mapping.service.DDMStructureVersionLocalService;
+import com.liferay.dynamic.data.mapping.util.DDMFormSerializeUtil;
+import com.liferay.portal.kernel.model.change.tracking.CTModel;
+import com.liferay.portal.kernel.service.change.tracking.CTService;
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.test.rule.Inject;
+
+import org.junit.runner.RunWith;
+
+/**
+ * @author Igor Costa
+ */
+@RunWith(Arquillian.class)
+public class DDMStructureVersionDDMStructureCTUpgradeProcessTest
+	extends DDMStructureCTUpgradeProcessTest {
```

Comment:
Sorry, I think I said the wrong class previously.
Can you extend from com.liferay.dynamic.data.mapping.upgrade.v5_1_1.test.
				DDMStructureVersionDDMStructureCTUpgradeProcessTest ?

Reply:
As an example, see com.liferay.dynamic.data.mapping.upgrade.v3_8_1.test.DDMStructureVersionDDMStructureCTUpgradeProcessTest